lnwire/your_peer_storage.go

```diff
@@ -0,0 +1,49 @@
+package lnwire
+
+import (
+	"bytes"
+	"io"
+)
+
+// YourPeerStorage is sent in response to a PeerStorage message. It contains
+// the data in the last PeerStorage message received by the peer. It is also
+// sent on reconnection to any peer we previously received PeerStorage from.
+type YourPeerStorage struct {
```

Comment:
just making a note here that I think this might change to just being a TLV on the `channel_reestablish` message as was done in the [initial](https://github.com/lightning/bolts/pull/881) proposal

Reply:
And then we would send an `AckPeerStorage` message with the ID that we sent along in the `PeerStorage` message (ie, will need to add an ID there too)

Reply:
I do not think that is how it is in the updated spec now